cmake/FindLibDRM.cmake

```diff
@@ -0,0 +1,50 @@
+# - Try to find libdrm
+#
+# LibDRM_FOUND - system has libdrm
+# LibDRM_LIBRARIES - Link these to use libdrm
+# LibDRM_INCLUDE_DIRS - the libdrm include dir
+
+# Copyright (c) 2021 The Khronos Group Inc.
+# Copyright (c) 2021 Valve Corporation
+# Copyright (c) 2021 LunarG, Inc.
+
+IF (NOT WIN32)
+
+  # use pkg-config to get the directories and then use these values
+  # in the FIND_PATH() and FIND_LIBRARY() calls
+  FIND_PACKAGE(PkgConfig)
```

Comment:
Dont need to use capital case for the keywords. `find_package` and `include` is just fine.